### PR TITLE
Deprecate EOL channels instructions

### PR DESCRIPTION
**Pull request by jonringer**

### CODE REVIEW

File: src/Release-Process-Walkthrough.md [hunk 1]
```diff
@@ -222,3 +222,8 @@ nix-shell -p "python3.withPackages (p: [ p.lxml p.requests p.pytest ])" -p cdrki
   1.  Someone with s3 buck permssions will need to run `https://github.com/NixOS/nixpkgs/blob/master/nixos/maintainers/scripts/ec2/create-amis.sh` (usually AmineChikhaoui).
 
   2.  [Create PR adding it to NixOS configuration](https://github.com/NixOS/nixpkgs/pull/101720).
+
+## A Month after Release
+
+1. [EOL Channels should be deprecated](https://github.com/NixOS/nixos-org-configurations/pull/201)
```
Comment (cole-h) [inline src/Release-Process-Walkthrough.md:228]: There's a difference between deprecated and unmaintained now -- 21.05 became deprecated immediately after 21.11 released, but was still technically maintained. A month after 21.11 released, 21.05 became unmaintained.

https://github.com/NixOS/nixos-org-configurations/pull/192

So basically, there should be another "post-release" step that sets the previous release as "deprecated" (but *not* "unmaintained"), in addition to this  "1 month after" section.

**Review by cole-h — CHANGES_REQUESTED** [review]

File: src/Release-Process-Walkthrough.md (hunk 1 above)
Comment (jonringer) [reply src/Release-Process-Walkthrough.md:228]: Yea, poor wording

Comment (jonringer) [reply src/Release-Process-Walkthrough.md]: added deprecation step at release time

reworded EOL step to mention it being "unmaintained" instead

**Review by cole-h — APPROVED** [review]
LGTM, thanks :)

**Review by mweinelt — APPROVED** [review]

**Pull request merged**